C/Wifi-Passwords/README.md

```diff
@@ -0,0 +1,16 @@
+# Saved Wifi Passwords
+A cross platform C program which queries and writes all saved WiFi networks and their passwords in a text file.
+
+## Executing the Program:
+* Compile the program: `gcc gen-wifi.c`.
+* On windows, do `a.exe`.
+* On Linux, do `./a.out`.
+
+**Contents of `wifi_password.txt`:**
+```txt
+<SSID 1>:<password>
+<SSID 2>:<password>
+.
```

Comment:
Add a sample use case /example of your code in readme file.

Reply:
@yashaswibiyahut done.